Eveneum.Tests/Eveneum.Tests.csproj

```diff
@@ -5,9 +5,6 @@
   <ItemGroup>
     <Content Remove="C:\Users\Jakub Konecki\.nuget\packages\microsoft.azure.cosmos\3.15.0\contentFiles\any\netstandard2.0\ThirdPartyNotice.txt" />
```

Comment:
eh? consider updating cosmos sdk ver - the issue of the spurios content has long been resolved